.gitlab-ci-check-docker-build.yml

```diff
@@ -28,10 +35,9 @@ build:
   services:
     - docker:19.03.5-dind
   before_script:
-    - export DOCKER_REPOSITORY="mendersoftware/${CI_PROJECT_NAME}"
     - export DOCKER_TAG=${CI_COMMIT_REF_SLUG:-master}
     - export SERVICE_IMAGE="$DOCKER_REPOSITORY:$DOCKER_TAG"
```

Comment:
I'm not sure if this would ever not apply, or if it is just searching for alignments... but might `"$DOCKER_REPOSITORY":"$DOCKER_TAG"` be necessary here as well - if the `COMMIT_TAG` is built similarly?

Reply:
It is not necessary as `: ` is not a valid character for bash variables while `_` is. However, you already know how much I love alignment, so I will quote these ones here also.